src/VisualStudio/IntegrationTest/New.IntegrationTests/CSharp/CSharpCodeActions.cs

```diff
@@ -18,6 +17,9 @@
 using Roslyn.VisualStudio.IntegrationTests;
 using Xunit;
 
+using CodeFixesFixAllScope = Microsoft.CodeAnalysis.CodeFixes.FixAllScope;
+using CodeRefactoringsFixAllScope = Microsoft.CodeAnalysis.CodeRefactorings.FixAllScope;
```

Comment:
i'm very skeptical we need separate types for these.  i am 100% ok with the CodeFixes type, and having CodeRefactorings just reference that.  it's not unclean or unclear to me.